### PR TITLE
DEVDOCS-4705: [new] BOPIS, add BOPIS webhooks

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-4705]

**DO NOT MERGE until [PR 159](https://github.com/bigcommerce/api-specs-ssot/pull/159) is merged** 

## What changed?
* Add links to BOPIS webhooks on main webhooks & channel webhooks page

## Anything Else?
Note that location webhooks will also be in API references [PR 1318](https://github.com/bigcommerce/api-specs/pull/1318)

[DEVDOCS-4705]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4705?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by kristinapototska — APPROVED** [review]

**Comment by bc-andreadao** [comment]
@kristinapototska Heads up, I removed the inventory webhooks from the Channels Webhooks doc (due to your comment in [PR 159 ](https://github.com/bigcommerce/api-specs-ssot/pull/159) that inventory and locations no nothing about channels

File: docs/api-docs/webhooks/webhook-events.mdx [hunk 1]
```diff
@@ -783,3 +791,4 @@ Consult the [themes section of the Channel Webhooks Guide](/api-docs/channels/gu
 * [Webhooks Tutorial](/api-docs/store-management/webhooks/tutorial)
```
Comment (bc-traciporter) [inline docs/api-docs/webhooks/webhook-events.mdx:791]:
```suggestion
* [Webhooks Tutorial](/api-docs/store-management/webhooks/setting-up-webhooks)
```

**Review by bc-traciporter — COMMENTED** [review]
I updated the link to the Webhooks Tutorial.

**Review by bc-traciporter — APPROVED** [review]
Great job!

**Pull request merged**